scripts/install/apt-packages-list.txt

```diff
@@ -0,0 +1,71 @@
+# This file contains a list of apt packages to install.
+# Lines starting with # are comments as are blank lines,
+# and # starts trailing comments.
+
+# dcraw decodes Canon raw images, and gphoto2 takes pictures with
+# Canon DSLRs, among others.
+dcraw
+gphoto2
+
+# Packages needed according to http://astrometry.net/doc/build.html
+libbz2-dev
+libbz2-dev
+libcfitsio-dev
+libjpeg-dev
+libnetpbm10-dev
+libpng12-dev
+netpbm
+python-dev
+python-numpy
+python-pyfits
+swig
+zlib1g-dev
+
```

Comment:
Looks like `byobu` comes with Ubuntu server but not desktop. Let's put it in here to make sure we have it. It will pull in `tmux` also (I think we should default to byobu but will make comments in #97 )

Reply:
Added byobu & tmux (explicitly).